_rules/table-headers-attribute-refer-to-data-cells-a25f45.md

```diff
@@ -405,7 +405,7 @@ The `table` is not [included in the accessibility tree][].
 This `table` doesn't have a role of `table`, `grid` or `treegrid`.
 
 ```html
-<table role="region">
+<table role="region" aria-label="Hello">
```

Comment:
I wonder if this isn't a slightly clearer example:

```suggestion
<table role="heading" aria-level="1">
```

Reply:
I'm okay with this change. We already have three approvals. If we apply this change, does it mean we need to request a review again? @WilcoFiers